src/main/kotlin/io/github/mojira/arisa/modules/RevokePriorityModule.kt

```diff
@@ -29,6 +58,16 @@ class RevokePriorityModule : Module {
         else -> "-1"
     }
 
+    private fun assertValidPriority(priorityId: String) =
+        if (isValidPriority(priorityId)) {
+            Unit.right()
+        } else {
+            OperationNotNeededModuleResponse.left()
+        }
+
+    private fun isValidPriority(priority: String): Boolean =
+        setOf("-1", "11700", "11701", "11702", "11703").contains(priority)
```

Comment:
To avoid duplication, what about having a map from priority name to ID, which is then used by `getId`, and `isValidPriority` could use the values of the map instead.